### ISSUE TITLE
data_dir not set

### ISSUE THREAD

**Issue by whikloj** [scossu/lakesuperior#110]
### Environment

Operating system:  OS X

Python version: 3.7.3

LAKEsuperior release, branch, or commit #: master

### Steps to reproduce

Ran:
1. `git clone --recurse-submodules https://github.com/scossu/lakesuperior.git`
1. `cd lakesuperior`
1. `docker-compose up`

### Observed behavior

Starts to come up then
```
Installing collected packages: lakesuperior
  Found existing installation: lakesuperior 1.0.0a19
    Uninstalling lakesuperior-1.0.0a19:
      Successfully uninstalled lakesuperior-1.0.0a19
  Running setup.py develop for lakesuperior
Successfully installed lakesuperior
You are using pip version 19.0.3, however version 19.2.2 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
2019-08-15 18:35:30,722 [MainThread] coilmq.start - INFO - Created server:<coilmq.server.socket_server.ThreadedStompServer object at 0x7f92fd0bbe10>
2019-08-15 18:35:30,722 [MainThread] coilmq.start - INFO - Stomp server listening on 127.0.0.1:61613
Reading configuration at /usr/local/lakesuperior/lakesuperior/etc.defaults
Traceback (most recent call last):
  File "/usr/local/bin/lsup-admin", line 11, in <module>
    load_entry_point('lakesuperior', 'console_scripts', 'lsup-admin')()
  File "/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py", line 489, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2793, in load_entry_point
    return ep.load()
  File "/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2411, in load
    return self.resolve()
  File "/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2417, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/local/lakesuperior/lakesuperior/lsup_admin.py", line 13, in <module>
    from lakesuperior.api import admin as admin_api
  File "/usr/local/lakesuperior/lakesuperior/api/admin.py", line 4, in <module>
    from lakesuperior.config_parser import parse_config
  File "/usr/local/lakesuperior/lakesuperior/config_parser.py", line 91, in <module>
    config = parse_config()
  File "/usr/local/lakesuperior/lakesuperior/config_parser.py", line 64, in parse_config
    if not _config['application']['data_dir']:
KeyError: 'data_dir'

Error: 'data_dir'
Reading configuration at /usr/local/lakesuperior/lakesuperior/etc.defaults
```

### Expected behavior

Container comes up at least enough that I can get into the container and fix the configuration.

### Other notes worth mentioning

(any other context details that may be affecting your outcome)

**Comment by scossu** [comment]
@whikloj This was fixed in 20d9102627fce3a2d3807b80887d8ee690ef74b5 which is still in the development branch. FYI I am working on a separate repository for Compose files using pre-built Docker images, while this one will contain a Docker file aimed at development purposes.

**Comment by scossu** [comment]
Fix merged into master.
